README.md

```diff
@@ -17,83 +17,318 @@ Main characteristics:
+
+### readable
+
+[api documentation](https://amadeusitgroup.github.io/tansu/functions/readable.html)
+
+Similar to Svelte stores, this function generates a store where the value cannot be externally modified.
+
+```typescript
+import {readable} from '@amadeus-it-group/tansu';
+
+const time = readable(new Date(), (set) => {
+  const interval = setInterval(() => {
+    set(new Date());
+  }, 1000);
+
+  return () => clearInterval(interval);
+});
+```
+
+### derived vs computed
+
+While derived and computed may appear similar, they exhibit distinct characteristics that can significantly impact effectiveness based on use-cases:
+
+- **Declaration of Dependencies:**
+  - `computed`: No explicit declaration of store dependencies is required, providing more flexibility in code composition.
+  - `derived`: Requires explicit declaration of dependencies.
+
+- **Performance:**
+  - `computed`: Better performance by re-running the function only based on changes in the stores involved in the last run.
+  - `derived`: Re-run the function each time a dependent store changes.
+
+- **Asynchronous State:**
+  - `computed`: Unable to manage asynchronous state.
+  - `derived`: Can handle asynchronous state with the `set` method.
+
+- **Skipping Value Emission:**
+  - `computed`: Does not provide a mechanism to skip emitting values.
+  - `derived`: Allows skipping the emission of values by choosing not to call the provided `set` method.
+
+- **Setup and Teardown:**
+  - `computed`: Lacks explicit setup and teardown methods.
+  - `derived`: Supports setup and teardown methods, allowing actions such as adding or removing DOM listeners.
+
+While `computed` feels more intuitive in many use-cases, `derived` excels in scenarios where `computed` falls short, particularly in managing asynchronous state and providing more granular control over value emissions.
+
+Carefully choosing between them based on specific requirements enhances the effectiveness of state management in your application.
+
+### Getting the value
+
+There are three ways for getting the value of a store:
+
+```typescript
+import {writable, get} from "@amadeus-it-group/tansu";
+
+const count$ = writable(1);
+const unsubscribe = count$.subscribe((count) => {
+  // Will be called with the updated value, synchronously first, then each time count$ changes
+  // must be unsubscribe to end the future calls.
+  console.log(count);
+});
+
+// A store is also a function that you can call to get the instant value.
+console.log(count$());
+
+// Equivalent to
+console.log(get(count$));
+```
+
+> [!NOTE]
+> Getting the instant value implies the subscription and unsubription on the store:
+>   - It can be important to know in case of setup/teardown functions.
+>   - In the same scope, prefer to store the value once in a local variable instead of calling `store$()` several times.
+>
+> `get` can work with Class variant of a store,
```

Comment:
```suggestion
> `get` can work with the class variant of a store.
>
```

Reply:
I'm going to remove it completely I think. The point is that, as @ExFlo mentioned it to me, the class usage is not mentioned in this part. It's a bit confusing. It's probably to be described in another section dedicated to the Tansu classes.